charts/minecraft/values.yaml

```diff
@@ -399,6 +399,8 @@ envFrom: []
 persistence:
   labels: {}
   annotations: {}
+  ## specify an extra volume to be mounted to /data instead of datadir.
+  # extraDataVolume: ""
```

Comment:
How about calling it "alternateDataVolumeName" (or "altDataVolumeName" to be shorter):

- "extra" implies "in addition to" but really this is replacing entirely the default data volume reference
- I propose adding the "Name" suffix to emphasize that is the name/reference to a pod volume and not the volume declaration in its entirety

Reply:
Great suggestion @itzg! I have updated the value name to `altDataVolumeName`. I agree that this name better reflects its intent. :+1: 

Thanks for the review!